docs/usage/runtime.md

```diff
@@ -40,6 +40,14 @@ require("babel").transform("code", { optional: ["runtime"] });
 $ babel --optional runtime script.js
 ```
 
+In Webpack:
+
+```js
+loaders: [
+  { test: /\.js$/, loaders: ['babel?optional=runtime']}
+]
+```
+
```

Comment:
I'd rather not put web pack in here. If we put every way to add this option it'd be really long.